packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -1,262 +1,13 @@
 import Ember from 'ember-metal/core';
 import EmberView from 'ember-views/views/view';
-import run from 'ember-metal/run_loop';
-import EmberObject from 'ember-runtime/system/object';
-import { set } from 'ember-metal/property_set';
 import compile from 'ember-template-compiler/system/compile';
 import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
 
 var view, lookup;
 var originalLookup = Ember.lookup;
 
-function testWithAs(moduleName, templateString, deprecated) {
-  QUnit.module(moduleName, {
-    setup() {
-      Ember.lookup = lookup = { Ember: Ember };
-
-      var template;
-      if (deprecated) {
-        expectDeprecation(function() {
-          template = compile(templateString);
-        }, 'Using {{with}} without block syntax (L1:C0) is deprecated. Please use standard block form (`{{#with foo as |bar|}}`) instead.');
-      } else {
-        template = compile(templateString);
-      }
-
-
-      view = EmberView.create({
-        template: template,
-        context: {
-          title: 'Señor Engineer',
-          person: { name: 'Tom Dale' }
-        }
-      });
-
-      runAppend(view);
-    },
-
-    teardown() {
-      runDestroy(view);
-      Ember.lookup = originalLookup;
-    }
-  });
-
-  QUnit.test('it should support #with-as syntax', function() {
-    equal(view.$().text(), 'Señor Engineer: Tom Dale', 'should be properly scoped');
-  });
-
-  QUnit.test('updating the context should update the alias', function() {
-    run(function() {
-      view.set('context.person', {
-        name: 'Yehuda Katz'
-      });
-    });
-
-    equal(view.$().text(), 'Señor Engineer: Yehuda Katz', 'should be properly scoped after updating');
-  });
```

Comment:
Replaced with `@test updating the context should update the alias` (although we lost the `title` part as I pointed out above. Thinking about it more, it seems good to preserve that part across all/most of the tests?

(Could write some small abstraction for that locally in that test class too. The way we are testing it here actually seems pretty good.)